WAVLTree/src/WAVLTree.java

```diff
@@ -16,7 +29,11 @@
    *
    */
   public boolean empty() {
-    return false; // to be replaced by student code
+      if (this.root == null) {
```

Comment:
need to check, instead of root == null, we can check if root is external node. Can be left for later

Reply:
Changed accordingly with !this.root.isInnerNode()